cfme/tests/configure/test_visual_cloud.py

```diff
@@ -145,10 +157,11 @@ def test_start_page(request, setup_a_provider, start_page):
     visual.login_page = start_page
     login.logout()
     login.login_admin()
-    level = re.split(r"\/", start_page)
-    assert menu.nav.is_page_active(None, level[0].strip(), level[1].strip()), "Landing Page Failed"
```

Comment:
Right now landing_pages is only 2 levels but it could be more. Plus I just thought it was ugly.